covalent_slurm_plugin/slurm.py

```diff
@@ -51,6 +51,12 @@
     "options": {
         "parsable": "",
     },
+    "srun_options": {
+        "slurmd-debug": "4",
```

Comment:
Can we switch the default to None here?  otherwise this will apply to all users' jobs.